tracing-appender/Cargo.toml

```diff
@@ -35,3 +36,6 @@ features = ["fmt", "std"]
 tracing = { path = "../tracing", version = "0.2" }
 time = { version = "0.3", default-features = false, features = ["formatting", "parsing"] }
 tempfile = "3"
+
+[features]
+compression = ["flate2"]
```

Comment:
also, does `flate2` have different feature flags for the various compression algorithms it provides? if so, we might want to have different feature flags for each compression algorithm, as well, so that we don't always enable _all_ the features of `flate2`...

Reply:
I changed te place of `[features]`.

I don't think looking at flate2 features that we can use any of their flags, Gzip and zlib are not behind any feature flag - at least the standard algorithms aren't. From flate2 docs:

```These three formats (DEFLATE, zlib, gzip) are all closely related and largely only differ in their headers/footers. This crate has three types in each submodule for dealing with these three formats.```